### ISSUE TITLE
[AXE-CORE]: Headings MUST not be empty

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#6371]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Headings **MUST** not be empty. See screenshot below. There is an empty `<h3>` which will be confusing or prevent users from accessing information on the page's structure.

### Why it Matters
Screen readers alert users to the presence of a heading tag. If the heading is empty or the text cannot be accessed, this could either confuse users or even prevent them from accessing information on the page's structure.

If the text inside a heading cannot be accessed by a screen reader, users of this technology will not be able to hear the contents of the heading. Since headings relay the structure of a webpage, it's crucial that users of screen readers are able to access the contents.

Applying heading markup (`<h1>` through `<h6>`) is a quick way to make text stand out, however, using it for anything other than headings will make navigating a web page more confusing for users of assistive technology.

In addition to making the page more accessible, headings have other benefits, since search engines use headings when filtering, ordering, and displaying results. Improving the accessibility of your site can also have the effect of making your page more findable.

In the same way that sighted users can glance at a page and get a sense of its contents, users of screen readers can do the same by navigating through headings. Well written and properly ordered headings can save users, especially those who use screen readers, a lot of time and frustration.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

 As a screen reader user, I want to clearly navigate the structure of the page by headings.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: production

## Steps to Recreate

1. Enter `https://www.va.gov/disability/after-you-file-claim/` in browser
2. Start screenreading device 
3. Navigate by headings
4. Validate the empty `<h3>`

## Possible Fixes (optional)

Remove the empty heading tag.

## WCAG or Vendor Guidance (optional)

* [Headings must not be empty](https://dequeuniversity.com/rules/axe/3.4/empty-heading?application=AxeChrome)

## Screenshots or Trace Logs

![vsa-pw-empty-h3](https://user-images.githubusercontent.com/57469/75703539-ab25ac00-5c85-11ea-90ee-eddb005084ff.png)

**Comment by RLHecht** [comment]
This change was made. There was an extra line space, which I deleted. Moving to validate and can close when it's live on the site later this afternoon.![Screen Shot 2020-03-16 at 9.41.41 AM.png](https://images.zenhubusercontent.com/5c9b7e302ac54336e2c327b8/3e649d67-42ea-4504-a099-93aa53b16c74)

**Comment by jenstrickland** [comment]
Thanks, @RLHecht ! Let me know when it's online and I'll validate, leave a comment, and close.

**Comment by MickinSahni** [comment]
2020-03-16 PW Standup
* Will validate once deploy goes out later today (Monday)

**Comment by RLHecht** [comment]
@jenstrickland This corrected page is now live. You can close out the ticket.

**Comment by jenstrickland** [comment]
@RLHecht The empty heading is still there. I emptied cache and did a hard load, tested a few times to be sure.

**Error message in [axe extension](http://bitly.com/aXe-Chrome)**
![Screen Shot 2020-03-17 at 9 28 36 AM](https://user-images.githubusercontent.com/57469/76860625-b950fc00-6831-11ea-907c-c325f2f86d79.png)

**Location of the empty heading**
![Screen Shot 2020-03-17 at 9 28 11 AM](https://user-images.githubusercontent.com/57469/76860626-b950fc00-6831-11ea-8c24-28824939d2ea.png)

**Comment by RLHecht** [comment]
@jenstrickland From the first screenshot above I thought the extra spacing was between the last two H2 questions on the page, which I removed. I'm not seeing any extra spacing in the Drupal page under "More information about disability ratings."

**Comment by jenstrickland** [comment]
@RLHecht I wonder about that extra space in the screenshot above, after the subway map, too, but it's not showing as an empty heading in the axe checker. It says the empty heading is in the "list" of items under More information about disability ratings. 

If you'd like to hop on a call and show me how this area is created, I have a suspicion where it is.

**Comment by jenstrickland** [comment]
- It appears this is an issue in the template, specifically the way the horizontal rule is implemented. 

- The line is created by an empty `<h3>` with CSS to add a gray border-bottom.

- Screenshots showing the issues included in this comment.

Maybe this could be changed to add a border-bottom to the `<h2>` or a border-top to the top of the `<ul>`, so that accessibility issues are avoided.

![Screen Shot 2020-03-17 at 11 20 57 AM](https://user-images.githubusercontent.com/57469/76872892-6da74e00-6843-11ea-8119-1e06de67f2a8.png)




![Screen Shot 2020-03-17 at 11 21 36 AM](https://user-images.githubusercontent.com/57469/76872903-71d36b80-6843-11ea-8acd-8620c6b60613.png)

**Comment by MickinSahni** [comment]
2020-03-18 PW Standup
* Pulling out of sprint so we can groom the front-end changes required to resolve the empty header

**Comment by MickinSahni** [comment]
2020-03-18 PW Grooming
* Assigning to nick/kelson to groom and determine a path forward
* randi's portion is complete however

**Comment by ncksllvn** [comment]
Fixed, will go live today.

**Comment by MickinSahni** [comment]
Adding to sprint 17!

**Comment by jenstrickland** [comment]
Fix validated. Thanks so much!
